src/user/supla_esp_devconn.c

```diff
@@ -1589,3 +1699,136 @@ void DEVCONN_ICACHE_FLASH supla_esp_calcfg_result(TDS_DeviceCalCfgResult *result
 	}
 }
 #endif /*BOARD_CALCFG*/
+
+#if defined(POWSENSOR2)
+void ICACHE_FLASH_ATTR supla_esp_em_extendedvalue_to_value(TElectricityMeter_ExtendedValue *ev, char *value) {
```

Comment:
Why do you copy the function from a different place?
https://github.com/SUPLA/supla-espressif-esp/blob/master/src/user/supla_esp_electricity_meter.c#L82